src/playback-watcher.js

```diff
@@ -157,16 +157,30 @@ export default class PlaybackWatcher {
     let seekable = this.seekable();
     let currentTime = this.tech_.currentTime();
 
-    if (this.tech_.seeking() &&
-        this.outsideOfSeekableWindow_(seekable, currentTime)) {
-      let seekableEnd = seekable.end(seekable.length - 1);
-
-      // sync to live point (if VOD, our seekable was updated and we're simply adjusting)
-      this.logger_(`Trying to seek outside of seekable at time ${currentTime} with ` +
-                   `seekable range ${Ranges.printableRange(seekable)}. Seeking to ` +
-                   `${seekableEnd}.`);
-      this.tech_.setCurrentTime(seekableEnd);
-      return true;
+    if (this.tech_.seeking()) {
+      if (this.afterSeekableWindow_(seekable, currentTime)) {
+        let seekableEnd = seekable.end(seekable.length - 1);
+
+        // sync to live point (if VOD, our seekable was updated and we're simply adjusting)
+        this.logger_(`Trying to seek outside of seekable at time ${currentTime} with ` +
```

Comment:
Since the much of this is the same, maybe just set a seekTime variable and do the log/seek at the end if it was set
```
let seekTime = NaN;
if (after) seekTime = seekable.end;
if (before) seekTime = seekable.start + 0.1;
if (!isNaN(seekTime)) log; setCurrentTime(seekTime);
```

Reply:
Yep will do